fdbserver/workloads/BackupCorrectness.actor.cpp

```diff
@@ -505,26 +504,6 @@ struct BackupAndRestoreCorrectnessWorkload : TestWorkload {
 			state Reference<IBackupContainer> lastBackupContainer =
 			    wait(BackupConfig(logUid).backupContainer().getD(cx));
 
-			// Occasionally start yet another backup that might still be running when we restore
```

Comment:
What's the purpose for this code originally? Removing this seems to suggest that we should disallow backup during a restore. If this is the case, then we might need to add such a logic (and keep this code to verify it)?

Reply:
To me, it looks like that we start this backup, do not wait it to finish, then start a restore (in the code below).

Reply:
The case to run a backup during a restore is covered in tests/fast/BackupCorrectnessClean.toml now (see backup3).